### PR TITLE
fix: Claim button is visible even after the claiming deadline passed

### LINKED ISSUE #2098: [BUG] Collectibles - Claim button is visible even after the claiming deadline passed.
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Product

Other

### Current Behavior

The claiming deadline is passed already, but collectibles page still shows “Claim” button.
image

### Expected Behavior

Once the claiming deadline is passed, "Claim" button is not shown on collectibles page.

### Steps To Reproduce

Go to collectibles page with the claiming right for NFT which claiming deadline is passed.

### Environment

```markdown
- Device: any
- OS: any
- Browser: any
- Wallet: any
```

### Anything else?

_No response_

### PR DESCRIPTION
**Pull request by memoyil**
Fixes #2098 

Not sure if this is the correct way, @ChefNina and @Chef-Chungus can you confirm?

To review: 

https://deploy-preview-2103--pancakeswap-dev.netlify.app/

### CODE REVIEW

**Review by hachiojidev — CHANGES_REQUESTED** [review]
If the deadline is passed then we can simply remove this component and use the base one.

**Comment by memoyil** [comment]
> If the deadline is passed then we can simply remove this component and use the base one.

Done

File: src/views/Collectibles/components/NftList.tsx [hunk 1]
```diff
@@ -6,21 +6,19 @@ import { fetchWalletNfts } from 'state/collectibles'
...
-  'easter-caker': EasterNftCard,
-  lottie: LotteryNftCard,
-  lucky: LotteryNftCard,
-  baller: LotteryNftCard,
+  'easter-storm': NftCard,
+  'easter-flipper': NftCard,
+  'easter-caker': NftCard,
+  lottie: NftCard,
+  lucky: NftCard,
+  baller: NftCard,
```
Comment (hachiojidev) [inline src/views/Collectibles/components/NftList.tsx:21]: Just remove these. It will use the default card in the render component.

**Review by hachiojidev — CHANGES_REQUESTED** [review]

File: src/views/Collectibles/components/NftList.tsx (hunk 1 above)
Comment (hachiojidev) [reply src/views/Collectibles/components/NftList.tsx:21]: Keep the empty object though

Comment (memoyil) [reply src/views/Collectibles/components/NftList.tsx]: Done

**Review by hachiojidev — APPROVED** [review]

**Pull request merged**